tapdb/multiverse.go

```diff
@@ -273,3 +273,70 @@ func (b *BaseMultiverse) RegisterIssuance(ctx context.Context,
 
 	return issuanceProof, nil
 }
+
+// RegisterBatchIssuance inserts a new minting leaf batch within the multiverse
+// tree and the universe tree that corresponds to the given base key(s).
+func (b *BaseMultiverse) RegisterBatchIssuance(ctx context.Context,
+	items []*universe.IssuanceItem) error {
+
+	insertProof := func(item *universe.IssuanceItem,
+		dbTx BaseMultiverseStore) error {
+
```

Comment:
I think re that question about idempotence, _this_ is where (or we'd add the initial check to skip if already present?